scss/application.scss

```diff
@@ -1,3 +1,4 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
 @import 'equalizer';
 @import 'detail';
+@import 'variables';
```

Comment:
Import order on .scss files matters, so move variables below equalizer and above detail, so detail can use the variables!